examples/overview-urdf.cpp

```diff
@@ -0,0 +1,18 @@
+#include "pinocchio/multibody/model.hpp"
+#include "pinocchio/parsers/urdf.hpp"
+#include "pinocchio/algorithm/kinematics.hpp"
+
+int main(int argc, char ** argv)
+{
+  std::string filename = (argc<=1) ? "ur5.urdf" : argv[1];
+  se3::Model model;
+  se3::urdf::buildModel(filename,model); 
+  se3::Data data(model);
+  Eigen::VectorXd q(model.nq); q    << 2, -.5, 1.8, 1.8, 2.6, -2;
```

Comment:
What happens if the model is not the same as ur5 here? I think we should initialize q with the neutral element.